PyQt5-stubs/QtWidgets.pyi

```diff
@@ -389,7 +389,8 @@ class QWidget(QtCore.QObject, QtGui.QPaintDevice):
     def setDisabled(self, a0: bool) -> None: ...
     def setEnabled(self, a0: bool) -> None: ...
     def isEnabledTo(self, a0: 'QWidget') -> bool: ...
-    def setStyle(self, a0: 'QStyle') -> None: ...
+    def setStyle(self, style: 'QStyle') -> None: ...
+    def setStyle(self, style: str) -> None: ... 
```

Comment:
I think this would fix the error in the build.

```suggestion
    @typing.overload
    def setStyle(self, style: 'QStyle') -> None: ...
    @typing.overload
    def setStyle(self, style: str) -> None: ... 
```

But...  I think this was applied to `QWidget` not `QApplication`?

Reply:
Would a `Union` not be more applicable here?

Reply:
Maybe?  Seems reasonable, I think.

Reply:
You guys are right. The Union is the way to go. Didn't notice the double setStyle definition at first...

Reply:
As @altendky mentioned, you're still modifying `QWidget.setStyle`, which [doesn't have any overloads](https://doc.qt.io/qt-5/qwidget.html#setStyle). You'll need to change `QApplication.setStyle`, which is what you linked and has the overloads.

Reply:
Actually, it already has that overload: 

https://github.com/stlehmann/PyQt5-stubs/blob/704207e90bee7b36ec9861dfa6b39f06a27c6718/PyQt5-stubs/QtWidgets.pyi#L1189-L1193

And I was wrong about using a `Union` because the return type differs between the two signatures.